### ISSUE TITLE
Unable to install Python binding on M1 Monterey macOS

### ISSUE THREAD

**Issue by tmokmss** [asg017/sqlite-vss#83]
Hi, when I try to install the [sqlite-vss package](https://pypi.org/project/sqlite-vss/) on my macOS (M1 Monterey) machine, I get an error:

```sh
$ pip install sqlite-vss
ERROR: Could not find a version that satisfies the requirement sqlite-vss (from versions: none)
ERROR: No matching distribution found for sqlite-vss
```

The problem is that the name of wheel for arm Mac is `sqlite_vss-0.1.1-py3-none-macosx_13_0_arm64.whl`, which cannot be installed on Monterey(=v12). When I downloaded the wheel and rename it, it was installed successfully.

```sh
$ mv sqlite_vss-0.1.1-py3-none-macosx_13_0_arm64.whl sqlite_vss-0.1.1-py3-none-macosx_12_0_arm64.whl
$ pip install sqlite_vss-0.1.1-py3-none-macosx_12_0_arm64.whl
# -> success
```

Is there any reason to set the least macOS version to 13 for the arm64 build? If not, I guess the wheel file name should be fixed.

**Comment by asg017** [comment]
I will rename the wheels to use `py3-none-macosx_12_0_arm64.whl` in the next release, thanks for letting me know! I just want to test and verify that it won't break those on Ventura before publishing

**Comment by tmokmss** [comment]
@asg017 I think `11_0` would be the best, as Big Sur(11) is the first version that supports M1 Macs ([ref](https://discussions.apple.com/thread/252075843)). Thanks!

**Comment by asg017** [comment]
@tmokmss just published `0.1.2-alpha.2` with a new wheel name - can you `pip install sqlite-vss==0.1.2a2` to see if it works for you?

The wheel is `sqlite_vss-0.1.2a2-py3-none-macosx_11_0_arm64.whl`

**Comment by tmokmss** [comment]
@asg017 I confirmed now I can install and use the package successfully. Thanks!
